Setup to upload or create new adjusting entries (#349)
monitr/docs: docs/setup/adjusting-entries/index.mdx

---
id: adjusting-entries
title: Adjusting entries
slug: ../adjusting-entries
---
import { Icon, ActionButton, PrimaryButton } from "@fluentui/react";
import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

import AdjustingEntryTabCSV from "./adjusting-entry-csv.mdx";
import AdjustingEntryTabUI from "./adjusting-entry-ui.mdx";
import BrowserFrame from "../../../src/components/atoms/BrowserFrame";
import adjustingEntryScreenshot from "./adjusting-entry-screenshot.png";

## Setup your adjusting entries

> You can setup your adjusting entries either by uploading a CSV file or via our user-interface.

Reviewer: ```suggestion
You can setup your adjusting entries either by uploading a CSV file or via our user interface.
```

The adjusting entries table contains the following fields:

- Description: Description of the adjusting entry.
- Period: Month of the adjusting entry.

> - Account: Code of the account used in the adjusting entry.

Reviewer: ```suggestion
- Account: Account number of the adjusting entry.
```

> - Analytical axis: Account and description of the selected analytical Code.

Reviewer: ```suggestion
- Analytical axis: Code and description of the selected analytical axis of the adjusting entry.
```

> - Amount: Total amount of the selected adjusting entry.

Reviewer: ```suggestion
- Amount: Amount of the adjusting entry.
```
